exchange/exchange_test.go

```diff
@@ -1196,7 +1196,7 @@ func TestCategoryDedupe(t *testing.T) {
 	cats4 := []string{"IAB1-2000"}
 	bid1 := openrtb.Bid{ID: "bid_id1", ImpID: "imp_id1", Price: 10.0000, Cat: cats1, W: 1, H: 1}
 	bid2 := openrtb.Bid{ID: "bid_id2", ImpID: "imp_id2", Price: 15.0000, Cat: cats2, W: 1, H: 1}
-	bid3 := openrtb.Bid{ID: "bid_id3", ImpID: "imp_id3", Price: 10.0000, Cat: cats1, W: 1, H: 1}
+	bid3 := openrtb.Bid{ID: "bid_id3", ImpID: "imp_id3", Price: 20.0000, Cat: cats1, W: 1, H: 1}
```

Comment:
It's unclear to me which parts of this are testing the `dupeKey = category` branch vs the `dupeKey = categoryDuration` branch. Consider separating into separate test cases for clarity.

Reply:
Unit tests added